__tests__/get-endpoints.test.js

```diff
@@ -142,3 +140,51 @@ test("Handles invalid endpoint", () => {
         });
 });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+    test("responds with an array of comments for the given article_id", () => {
+        return request(app)
+            .get("/api/articles/1/comments")
+            .expect(200)
+            .then((result) => {
+                const comments = result.body;
+                expect(Array.isArray(comments)).toBe(true);
+                expect(Object.keys(comments[0])).toHaveLength(6);
+                expect(comments).toBeSortedBy("created_at", { descending: true });
+                comments.forEach((comment) => {
+                    expect(comment).toHaveProperty("comment_id", expect.any(Number));
+                    expect(comment).toHaveProperty("body", expect.any(String));
+                    expect(comment).toHaveProperty("article_id", expect.any(Number));
+                    expect(comment).toHaveProperty("author", expect.any(String));
+                    expect(comment).toHaveProperty("votes", expect.any(Number));
+                    expect(comment).toHaveProperty("created_at", expect.any(String));
+                });
+            });
+    })
+
+    test("Respond with custom error when an article exists but there is no comments for that article", () => {
```

Comment:
Really good to have this specific condition considered, however I would argue that in this case this actually isn't an error as nothing has gone wrong. It's a valid article but there just happens to be no comments on it as yet. So a 200 would be appropriate :)